src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,25 @@
 import React from 'react';
+import { CommentInfo } from '../CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
-);
+import { Comments } from '../../types/Comments';
```

Comment:
Rename the imported type 'Comments' to 'Comment' for better readability and consistency.